mspelman2389.md

```diff
@@ -0,0 +1,4 @@
+# mspelman2389
+
```

Comment:
@prashantkad I included the weather conditions, which are warm and sunny. Good now?